README.md

```diff
@@ -39,51 +43,8 @@ yarn add @segment/analytics-next
 pnpm add @segment/analytics-next
 ```
 
-2. Import the package into your project and you're good to go (with working types)! Example react app:
+2. Import the package into your project and you're good to go (with working types)! 
```

Comment:
I think the intent of the sample here was to show what the API looks like to load analytics when using the package directly, we just happened to use react as the example.

If we link to the react-example section, maybe this snippet can be reduced to something like:

```ts
import { Analytics, AnalyticsBrowser, Context } from '@segment/analytics-next'

async function loadAnalytics(): Promise<Analytics> {
  const [ analytics, context ] = await AnalyticsBrowser.load({ writeKey })
  return analytics
}
```